planning/mission_planner/schema/mission_planner_nodes.shema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Mission Planner nodes",
+  "type": "object",
+  "definitions": {
+    "mission_planner_nodes": {
+      "type": "object",
+      "properties": {
+        "map_frame": {
+          "type": "string",
+          "description": "The frame name for map ",
+          "default": "map"
+        },
+        "arrival_check_angle_deg": {
+          "type": "number",
+          "description": "Angle threshold for goal check ",
+          "default": "45.0"
+        },
+        "arrival_check_distance": {
+          "type": "number",
+          "description": "Distance threshold for goal check",
+          "default": "1.0",
+          "minimum": "0.0"
```

Comment:
The bounds value shouldn't be within quotation marks. This was incorrect in the coding guidelines, my bad! I've updated the coding guidelines accordingly.
* https://github.com/autowarefoundation/autoware-documentation/pull/434